exchanges/ftx/ftx.go

```diff
@@ -1235,15 +1241,19 @@ func (f *FTX) compatibleOrderVars(orderSide, orderStatus, orderType string, amou
 	case strings.ToLower(order.Open.String()):
 		resp.Status = order.Open
 	case closedStatus:
-		if filledAmount != 0 && filledAmount != amount {
-			resp.Status = order.PartiallyCancelled
-			break
-		}
-		if filledAmount == 0 {
+		switch {
+		case filledAmount <= 0:
+			// Order is closed with a filled amount of 0, which means it's
+			// cancelled.
 			resp.Status = order.Cancelled
-			break
-		}
-		if filledAmount == amount {
+		case math.Abs(filledAmount-amount) > 1e-6:
```

Comment:
NOTE: The smallest minProvideSize I could find is "minProvideSize":0.00001 so this looks good!